tuna/worker_interface.py

```diff
@@ -406,28 +406,34 @@ def queue_end_reset(self):
 
   def load_job_queue(self, session, ids):
     """load job_queue with info for job ids"""
+    # pylint: disable=comparison-with-callable
```

Comment:
Will this need to be disabled every time we use a sqlalchemy join?

Reply:
Only when we use session or solver - elements that have the decorator. Pylint does not recognize them yet, many people seem to be complaining about this :) maybe soon